simulator/examples/text-custom-font.rs

```diff
@@ -29,19 +31,21 @@ fn main() -> Result<(), core::convert::Infallible> {
     let position = Point::new(27, 22);
     let row_offset = Point::new(0, 44);
 
-    egtext!(
-        text = "123",
-        top_left = position,
-        style = text_style!(font = SevenSegmentFont, text_color = BinaryColor::On,)
-    )
-    .draw(&mut display)?;
-
-    egtext!(
-        text = "456",
-        top_left = position + row_offset,
-        style = text_style!(font = SevenSegmentFont, text_color = BinaryColor::On,)
-    )
-    .draw(&mut display)?;
+    Text::new("123", position)
+        .into_styled(
+            TextStyleBuilder::new(SevenSegmentFont)
+                .text_color(BinaryColor::On)
+                .build(),
+        )
+        .draw(&mut display)?;
+
+    Text::new("456", position + row_offset)
+        .into_styled(
+            TextStyleBuilder::new(SevenSegmentFont)
+                .text_color(BinaryColor::On)
+                .build(),
```

Comment:
Creating a variable to share the text style between calls would make the code cleaner IMO, otherwise:
```suggestion
            TextStyle::new(SevenSegmentFont, BinaryColor::On)
```